rocks-db/src/dump_client.rs

```diff
@@ -83,6 +83,16 @@ struct AssetRecord {
 }
 
 impl Storage {
+    /// It concurrently creates a `CSV file` for each provided file.
+    /// This separation into files makes sense because that each file corresponds to a separate table in `Postgres`.
+    ///
+    /// # Args:
+    /// `metadata_file_and_path` - The file and path corresponding to the metadata data.
```

Comment:
I think it would be better to indicate which table in Postgre data corresponds to. For example metadata corresponds to 'tasks' table